runtime/etiao/src/main/java/quarks/runtime/etiao/EtiaoJob.java

```diff
@@ -51,6 +53,10 @@ public EtiaoJob(DirectGraph graph, String topologyName, ServiceContainer contain
         ControlService cs = container.getService(ControlService.class);
         if (cs != null)
             cs.registerControl(JobMXBean.TYPE, getId(), getName(), JobMXBean.class, new EtiaoJobBean(this));
+        
+        this.jobs = (JobRegistry) container.getService(JobRegistryService.class);
```

Comment:
I considered the following alternatives:
   a) JobRegistry is an implementation class, then the runtime should register the registry, rather than the application. 
   b) JobRegistry has nothing specific to the etaio runtime, can be in its own module at the same level with JSONControl and JMXControl services. The JobRegistryService interface provides an add() method and is part of the same module.

I think (b) has the advantage of modularity.